### ISSUE TITLE
Add columns for lexeme and cognacy justifications:  [bibliographical reference] and [page number]

### ISSUE THREAD

**Issue by cormacanderson** [lingdb/CoBL#53]
### Clarification:  Two Separate Issues, in fact!

To clarify this issue, which was clear as mud in the original issue title, without any description!  This is actually **a mix-up of two different types of things called 'source' or 'reference', and which need to go on different forms**.  

There is also a big problem with the use of the term 'source' in a linguistic sense too, as in a source language for a cognate class.  So we need to kill use of the ambiguous term 'source', and in this case start talking about bibliographical reference.

The issue is clearer if you look at:  http://www.cobl.info/lexeme/28829/, and note the *_two *_ fields there both called 'source'.  Which from now on we should actually distinguish by different names.  
### 1. [Lexeme Justification Reference] to be abbreviated [Lex Rfc]

On http://www.cobl.info/lexeme/28829/ there is a section 'Source of lexical data' with a 'Source' field.  This is the bibliographical reference used to justify the data entered on this lexeme, such as its phonetic transcription, attestations of it in an old language now extinct, and so on.  It has also been used by some authors to justify their selection of this lexeme as the Swadesh term in that language.  That is really a judgement that we look to our branch experts to make on their own authority, although they should still feel free to cite a reference to justify their selection.

This [Lex Rfc] field (as it should be renamed on the table views) should appear as a column in the table views:  
In [1 Language All Meanings] view:  
In [1 Meaning All Languages] view:
In both cases it should go immediately before the [Root Form] column.

Also in both cases it should also be accompanied (followed) by another field, which is a page number.   This can already be seen when you try to edit the source, through a page like:  
http://www.cobl.info/lexeme/28829/edit-citation/36237/
### 2. [Cognacy Justification Reference] to be abbreviated [Cog Rfc]

This one seems like it's pretty much the same issue as:
https://github.com/lingdb/CoBL/issues/95
But I will describe it here in any case, to clarify the relationship between these two.

On http://www.cobl.info/lexeme/28829/ there is also a section 'Cognate coding' which also has a 'Source'.  This is the bibliographical reference used to justify the assignment of this lexeme to a cognate set. 

This [Cog Rfc] field (as it should be renamed on the table views) should appear as a column in:
the table view [1 Meaning All Cognate Sets]
It should go as the last column in the table.
(One might even wonder whether it could be useful on other forms too, but that is not a priority, and they are cramped enough anyway.)

**Comment by runjak** [comment]
Is this the same as issue #59, or is the issue here about some different fields?
We have `author` and `reviewer` fields now - so if it's the same, this issue could be closed.

**Comment by cormacanderson** [comment]
Not the same thing actually. Author and reviewer describe the people who will decide on and enter, and check the data, respectively. Sources / references refers to written sources that we will eventually link to - dictionaries and the like.

**Comment by runjak** [comment]
Ok, thanks for clarifying. :+1:

**Comment by cormacanderson** [comment]
While the clarification is good, this is not the same as https://github.com/lingdb/CoBL/issues/95. [Cog Rfc] above refers to the cognate judgement and is thus a property of the lexeme, i.e. lexeme x is a member of cognate set y, while what is referred to there is rather [Cog Set Rfc] which defines the cognate set as a whole. These will often overlap, but can sometimes be different: [Cog Rfc] is more likely to be a language or branch specific work, e.g. an Armenian etymological dictionary, while [Cog Set Rfc] is more likely to be a general work, such as LIV.
It should be noted that these fields are already in the database, but are not currently accessible on any html form. One can find [Cog Rfc] in the records of an individual lexeme (like also [Lex Rfc]), while [Cog Set Rfc] is found on the page of an individual cognate set. At the moment, it is possible to have (I think) an indefinte number of references of each type, although in practice I don't think I have come across more than three or four.

**Comment by cormacanderson** [comment]
From email exchange with @PaulHeggarty and Leszek on 17.10.2015:
It seems to me that there are several different types of reference works which are relevant here.

Firstly we have lexical sources: these are to give more info people who want to find out more about the semantics of the languages concerned. They are references which are relevant to the lexical entries, e.g. the OED, RAE, Académie Française, Duden etc. However, we have to realise that sometimes the best reference works available are bilingual dictionaries. This is the case even for Irish, which is hardly the worst described language in the list. Here, the standard reference works are Ó Domhnaill's Irish-English dictionary and de Bhaldraithe's English-Irish one. Both of them, especially Ó Domhnaill, are well put together, reliable sources, a long way from the Collins gem. Older, and often less reliable sources, are often the same. It's also the same for Scottish Gaelic and for Breton (maybe not for Welsh). For older languages, there just aren't reliable monolingual sources, for obvious reasons. Here, the best resources are often also bilingual with one or other European language.

Secondly, we have what we might call Swadesh sources. These are generally experts for the individual languages. However, when we are dealing with older languages, those experts might wish to refer to various works to justify the choice of word, clarify the semantics, or justify the inclusion of two rather than one word on rare occasions. Sometimes there are corpus studies available, but more often these sources are the lexical ones listed above. With Old Irish I always started with the dictionary (eDIL) but in problematic cases sometimes looked in the original texts to see how and where the word was being used, trying to get an idea of register, nuances etc.

Thirdly, we have cognacy sources. These are generally etymological, branch level, works. But sometimes they might be more specific - we have a Gaelic etymological dictionary which isn't bad and then there are works like the OED. For Old Irish, the early work of Vendryes is still a key source. Overlying these are more general works, like Britta's book on nouns or the LIV for the verbs, etc.

To an extent, we might want to consider the legacy sources which Leszek refers to, although how much DKB will be left in the database when we have finished with it I doubt. It's the old question of constantly repairing or patching something and at the end there might not be anything original left.

**Comment by leszekjocz** [comment]
As I remember, what did I mean with the "legacy sources", were not written sources but the work of the people that edited the database before me.
Although there were hunderts or even thousands of mistakes of every possible kind (from misspelings to completely false lexeme choice and shameful cognate coding), I did not have to change absolutely everything and to a greater or lesser degree I used the work of these people. In my opinion they should be somehow mentioned. I do not feel at all as the only author of the lists I worked on.
